tests/sentry/integrations/github/test_webhooks.py

```diff
@@ -68,10 +69,37 @@ def test_invalid_signature_event(self):
 
 @region_silo_test(stable=True)
 class PushEventWebhookTest(APITestCase):
-    @patch("sentry.integrations.github.client.get_jwt")
-    def test_simple(self, mock_get_jwt):
-        mock_get_jwt.return_value = ""
+    base_url = "https://api.github.com"
 
+    def _setup_repo_test(self, project):
```

Comment:
I noticed sometimes you create a second org and it takes up a lot of lines. Can this function create a variable amount of orgs, something like:
```suggestion
    def _setup_repo_test(self, project: Project, organizations: List[Organizations]) -> None:
```

Reply:
sometimes we attach both orgs to the same integration and other times we have separate integrations for each org